HaloDAO/halo-rewards
# remove hardhat folder

## Description of Changes



This quick and dirty PR just removes the `./contracts/hardhat` folder since in the previous merge, I moved its contents up to project root but forgot to delete the folder 🤦

No link to Jira ticket

## How To Test
- clone this repo
- run `npm i` at project root
- run `npx hardhat test` at project root, 42 tests should pass

### Developer Checklist:


### Reviewers Checklist:
